### ISSUE TITLE
Fail on error

### ISSUE THREAD

**Issue by brettstack** [swimlane-contrib/systemjs-route-bundler#37]
How do I go about failing the build if an error occurs during bundling (eg. cannot find a file (ENOENT))?

**Comment by marjan-georgiev** [comment]
We would probably need to reject the promises when an error is raised. At which stage of the process would you like to fail the build? Anywhere?

**Comment by brettstack** [comment]
Yes, anywhere on first error.

**Comment by marjan-georgiev** [comment]
Makes sense. There isn't really a point in continuing the build after an error occurs.
